python/secp256k1lab/pyproject.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "secp256k1lab"
+version = "0.1.0"
+description = "A library for prototypes using the secp256k1 curve"
+readme = "README.md"
+authors = [
+    { name = "Tim Ruffing", email = "crypto@timruffing.de" },
+    { name = "Jonas Nick", email = "jonasd.nick@gmail.com" },
+    { name = "Sebastian Falbesoner", email = "sebastian.falbesoner@gmail.com" }
+]
```

I think it will be fair to list @sipa as an author:
```suggestion
authors = [
    { name = "Pieter Wuille", email = "pieter@wuille.net" }, 
    { name = "Tim Ruffing", email = "me@real-or-random.org" },
    { name = "Jonas Nick", email = "jonasd.nick@gmail.com" },
    { name = "Sebastian Falbesoner", email = "sebastian.falbesoner@gmail.com" }
]
maintainers = [
    { name = "Tim Ruffing", email = "me@real-or-random.org" },
    { name = "Jonas Nick", email = "jonasd.nick@gmail.com" },
    { name = "Sebastian Falbesoner", email = "sebastian.falbesoner@gmail.com" }
]
```

based on:
https://github.com/bitcoin/bitcoin/commits/e486597f9a57903600656fb5106858941885852f/test/functional/test_framework/crypto/secp256k1.py

---

You've added `authors` but not `maintainers`. Was this intentional? (I also don't like the duplication too much, but well, it's correct...)